client/src/layout/Footer/Footer.js

```diff
@@ -1,7 +1,272 @@
+      gap: '30px',
+    },
+  },
+});
+
+const OurAwards = styled('div')({
+  '& > h3': {
+    paddingBottom: '30px',
+    color: 'white',
+  },
+  '& > p': {
+    paddingBottom: '30px',
+    color: 'white',
+  },
+});
+
+const ContactsData = styled('div')({
+  '& > h3': {
+    color: 'white',
+  },
+});
+
+const ContactsInfo = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  gap: '10px',
+  paddingTop: '30px',
+  '& > div': {
+    display: 'flex',
+    '& > a': {
+      textDecoration: 'none',
+      color: 'inherit',
+      marginLeft: '10px',
+    },
+    '& > p': {
+      marginLeft: '10px',
+      marginBottom: '20px',
+      color: 'white',
+    },
+  },
+});
+
+const SocialIcons = styled('div')({
+  display: 'flex',
+  justifyContent: 'left',
+  gap: '15px',
+  paddingTop: '20px',
+  paddingBottom: '40px',
+  '& > div': {
+    '& > svg': {
+      width: '40px',
+      height: '40px',
+      cursor: 'pointer',
+    },
+  },
+});
+
+const RecentTrips = styled('div')({
+  display: 'grid',
+  columnGap: '5px',
+  gap: '10px',
+  gridTemplateColumns: 'repeat(3, 1fr)',
+  marginBottom: '30px',
+  paddingTop: '30px',
+  '& > a >  img': {
+    width: '100%',
+    borderRadius: '5px',
+  },
+  '@media (min-width: 768px)': {
+    gridTemplateColumns: 'repeat(2, 1fr)',
+  },
+  '@media (min-width: 950px)': {
+    gridTemplateColumns: 'repeat(3, 1fr)',
+  },
+});
+
+const FooterNavigation = styled('div')({
+  borderTop: '1px solid rgba($color: #ffffff, $alpha: 0.5)',
+  margin: '0 auto',
+  padding: '30px',
+  maxWidth: '1200px',
+  '@media (min-width: 768px)': {
+    display: 'flex',
+    flexDirection: 'row-reverse',
+    justifyContent: 'space-between',
+    '& > *': { display: 'flex', gap: '30px' },
+  },
+});
+
+const NavigationList = styled('ul')({
+  display: 'flex',
+  justifyContent: 'left',
+  alignItems: 'center',
+  gap: '20px',
+  padding: '0',
+  paddingBottom: '20px',
+  '& > li': {
+    listStyleType: 'none',
+    ' & > a': {
+      textDecoration: 'none',
+      color: 'white',
+      '&:hover': {
+        color: 'blue',
+      },
+    },
+  },
+  '@media *min-width: 768px': {
+    paddingBottom: '0',
+  },
+});
+
+const Footer = () => (
+  <FooterContainer>
+    <FooterWrapper>
+      <Box>
+        <OurAwards>
+          <Typography variant="h3">Our Awards</Typography>
+          <Typography>
+            Ukraine is a country of brave people and countless opportunities. Here you will be accepted as a relative,
+            surrounded by care as at home and get everything done as one would do it for themselves
+          </Typography>
+          <Typography
+            sx={{ width: '250px', marginTop: '30px', paddingBottom: '40px' }}
+            component="img"
+            src="./assets/images/footer/awards.png"
+            alt="awards_mage"
+          />
+        </OurAwards>
+        <ContactsData>
+          <Typography variant="h3">Contact info</Typography>
```

Comment:
very complicated for readability. you could just use the below instead:

```suggestion
        <Box>
          <Typography variant="h3" sx={ { color: primary.contrastText } }>Contact info</Typography>
```

Reply:
solved